test/plugins/collections.test.js

```diff
@@ -20,6 +23,31 @@ const getCollectionMock = (collection) => {
     return mock;
 };
 
+const getCollectionsMock = (collections) => {
+    if (Array.isArray(collections)) {
+        return collections.map(getCollectionMock);
+    }
+
+    return getCollectionMock(collections);
+};
+
+// Get the mock pipeline in testPipelines using the input id
+const getPipelineMockFromId = (id) => {
+    let result = null;
+
+    testPipelines.forEach((pipeline) => {
+        if (pipeline.id === id) {
+            result = hoek.clone(pipeline);
+
+            result.update = sinon.stub();
+            result.toJson = sinon.stub().returns(pipeline);
+            result.remove = sinon.stub();
+        }
```

Comment:
could we return `Promise.resolve` here instead of setting the variable in the loop and then returning it afterwards?